openid-4-verifiable-presentations-1_0.md

```diff
@@ -460,6 +488,39 @@ To use `client_id_scheme` values `entity_id`, `did`, `verifier_attestation`, `x5
 
 Other specifications can define further values for the `client_id_scheme` parameter. It is RECOMMENDED to use collision-resistant names for such values.
 
+## Request URI Method POST {#request_uri_method_post}
+
+This request is offered at the Request URI endpoint by the Verifier. In case of success, the response is a Request Object that the Wallet MUST process in the same way as a Request Object as defined in [@RFC9101]. 
+
+The request MUST use the HTTP POST method with the https scheme and the media type set to "application/oauth-authz-req+jwt".
+
+The following parameters are defined: 
+
+`state`:
+: OPTIONAL. A JSON String containing the value of the corresponding authorization Request's `state` parameter, if present.
+
+`issuer`:
+: OPTIONAL. A JSON containing an HTTPS URL designating the Issuer URL of the Wallet (acting as a OAuth 2.0 Authorization Server). The Verifier MAY obtain the Wallet's metadata by adding the well-know location `oauth-authorization-server` as specified in [@!RFC8414]. Metadata MAY also be provided by other means, for example in the `wallet_metadata`parameter`.
+
+`wallet_metadata`:
+: OPTIONAL. A JSON Object containing metadata parameters as defined in (#as_metadata_parameters).
+
+`issuer_nonce`:
```

Comment:
why is this called an issuer_nonce? Isn't it provided by the wallet?

Reply:
issuer (AS) == wallet

Reply:
should we better call it `wallet_nonce`?